boot.php

```diff
@@ -2303,7 +2278,8 @@ function explode_querystring($query) {
 	}
 
 	$args = explode('&', substr($query, $arg_st));
-	foreach ($args as $k=>$arg) {
+	foreach ($args as $k => $arg) {
+		/// @TODO really compare type-safe here?
```

Comment:
Well, `explode` returns an array of strings no matter what, and `===` is faster than `==`, so you better use it whenever you can.